samples/bluetooth/throughput/sysbuild/hci_rpmsg/prj.conf

```diff
@@ -0,0 +1,32 @@
+CONFIG_IPC_SERVICE=y
```

Comment:
Wouldn't be better to create just configuration overlay file instead of introducing full configuration file in every sample that needs to modify other core image?

Reply:
A feature of sysbuild is not having the underlying image silently change your configuration which is something that happens with child/parent image

Reply:
Seems like there is option to overlay Konfig fragment using sysbuild:

> In the folder of the main application, create a Kconfig fragment or a devicetree overlay under a sysbuild folder, where the name of the file is <image>.conf or <image>.overlay, for example if your main application includes my_sample then create a sysbuild/my_sample.conf file or a devicetree overlay sysbuild/my_sample.overlay.

Having multiple copy of the same file will be a nightmare in maintenance.

Reply:
This is not supported when using board overlay, hence why the files are copied because the APPLICATION_CONFIG_DIR becomes that `sysbuild/app` folder

> Having multiple copy of the same file will be a nightmare in maintenance.

Having configurations change without the user being aware is a nightmare in updating modules. But yes, maintenance burden will be much higher